client/src/components/ChooseGenres.js

```diff
@@ -54,13 +56,42 @@ function ChooseGenres() {
     return <span>Error: {error.message}</span>;
   }
 
+  const handleClick = (event) => {
+    event.preventDefault();
```

Comment:
is there a form around those `button` elements?

Reply:
not at the moment